common/queue/src/main/java/org/thingsboard/server/queue/discovery/ZkDiscoveryService.java

```diff
@@ -303,6 +334,7 @@ public void childEvent(CuratorFramework curatorFramework, PathChildrenCacheEvent
      * Synchronized to ensure that other servers info is up to date
      * */
     synchronized void recalculatePartitions() {
+        delayedTasks.clear();
```

Comment:
good point to clean as repartition started, but futures from map not canceled and sits in scheduler